### ISSUE TITLE
GitHub for VCS to do ORM

### ISSUE THREAD

**Issue by albertyckwok** [oracle/learning-library#3008]
Hi, I am wondering if we can add GitHub in additional to GitLab for the optional lab of [Resource Manager](https://github.com/oracle/learning-library/blob/33140ed9950e1708c9b1e672495e183d3fd5ec31/oci-library/oci-hol/resource-manager/resource-manager/resource-manager.md). GitHub is publicly available without user to install and configure their own Gitlab to make it easier for participant to run the lab with less dependence.

Thanks very much in advance
Albert

**Comment by oegentil** [comment]
yes! it is definitely possible and I agree that github seems more popular.

I can't promise a date but it will be on my list of enhancements.

Thanks for the feedback!

**Comment by lucassrg** [comment]
Hey @albertyckwok , Oracle recently launched OCI DevOps service that can do Continuos Deployment to Oracle Developer services like OKE, Functions and Compute.
We also published a workshop that creates a CD pipeline using GitHub to provision the infrastructure through ORM. Please check if this is what you are looking for:
https://apexapps.oracle.com/pls/apex/dbpm/r/livelabs/view-workshop?wid=857

This lab will be presented next week during Oracle DevLive event with many other opportunities of learning:
https://developer.oracle.com/developer-live/simplifying-modern-app-dev
